[wjacobw] Ip

wjacobw opened the pull request:
# A first-level heading
## A second-level heading
### A third-level heading
This site was built using [GitHub Pages](https://pages.github.com/)
![Screenshot of a comment on a GitHub issue showing an image, added in the Markdown, of an Octocat smiling and raising a tentacle.](https://myoctocat.com/assets/images/base-octocat.svg)
- [x] #739
- [ ] https://github.com/octo-org/octo-repo/issues/740
- [ ] Add delight to the experience when all tasks are complete :tada:
- [x] Here is a simple footnote[^1].

A footnote can also have multiple lines[^2].

[^1]: My reference.
[^2]: To add line breaks within a footnote, prefix new lines with 2 spaces.
  This is a second line.

Commit:
Add Gradle support

Eclipse-Dominator committed:
Bump gradle and lib version

wjacobw committed:
v1

wjacobw committed:
newfile added

wjacobw committed:
newnewfile

wjacobw committed:
newnewnewfile

wjacobw committed:
Merge branch 'feature1'

wjacobw committed:
all the features on Level-7 except catching exception

wjacobw committed:
Features for level 8

wjacobw committed:
added features level 8

wjacobw committed:
add packages

wjacobw committed:
Resolve merge conflict in deadline.java and event.java

wjacobw committed:
rename all the file

wjacobw committed:
Merge branch 'add-gradle-support'

wjacobw committed:
Merge pull request #1 from wjacobw/add-gradle-support

Add gradle support

wjacobw committed:
put files in package

wjacobw committed:
add A-JUnit

wjacobw committed:
edit file path for jar

wjacobw committed:
add find method

wjacobw committed:
add java docs

wjacobw committed:
Merge pull request #4 from wjacobw/branch-Level-9

[wjacobw] iP

wjacobw committed:
Merge pull request #2 from wjacobw/branch-A-CodingStandard

[wjacobw] iP

wjacobw committed:
add GUI

wjacobw committed:
change build.gradle

andrechuakj commented on src/main/java/duke/Event.java:
```diff
@@ -0,0 +1,64 @@
+package duke;
+import java.time.*;
+import java.time.format.*;
```
Would importing classes explicitly be clearer?

andrechuakj commented on src/main/java/duke/Duke.java:
```diff
@@ -0,0 +1,42 @@
+import javafx.stage.Stage;
+
+
+public class Duke extends Application{
```
Maybe adding a whitespace before '{' would be clearer?

andrechuakj commented on src/main/java/duke/Task.java:
```diff
@@ -0,0 +1,70 @@
+     *
+     * @return true if the task is done, false otherwise.
+     */
+    public boolean idDone() {
```
Could this be a spelling mistake? (idDone instead of isDone)

andrechuakj commented on src/main/java/duke/TaskList.java:
```diff
@@ -0,0 +1,460 @@
+            }
+        }
+
+        else if (parser.getArr()[0].equals("find")) {
```
Would the format  ```} else if (parser.getArr()[0].equals("find")) {``` be clearer?

andrechuakj commented on src/main/java/duke/TaskList.java:
```diff
@@ -0,0 +1,460 @@
+            ui.matchingList(matchingList);
+            Answer();
+
+        }
+        else if (parser.getArr()[0].equals("event")) {
```
Similar to above, would ```} else if (parser.getArr()[0].equals("event")) {``` be clearer?

andrechuakj commented on src/main/java/duke/TaskList.java:
```diff
@@ -0,0 +1,460 @@
+            //appendToFile(zenithData, str);
+            refreshData();
+            Answer();
+        }
+        else if (parser.getArr()[0].equals("todo")){
```
Same as above

andrechuakj commented on src/main/java/duke/TaskList.java:
```diff
@@ -0,0 +1,460 @@
+                ui.specify();
+                Answer();
+            }
+        }
+        else if(parser.getArr()[0].isEmpty()) {
```
Same as above. Maybe consider using switch statements to make the code here more concise?

andrechuakj reviewed (commented):
LGTM. Just some nits to fix.

lambraydon commented on src/main/java/duke/Ui.java:
```diff
@@ -0,0 +1,161 @@
+     *
+     * @param list The matching list of tasks to display.
+     */
+    public void matchingList(List<Task> list) {
```
Perhaps it might be better to name this in the verb form? e.g displayMathchingList

lambraydon commented on src/main/java/duke/LoadFile.java:
```diff
@@ -0,0 +1,58 @@
+     */
+    public void load() throws FileNotFoundException {
+
+        File f = new File(getFilepath());
+        this.file = f;
+        Scanner s = new Scanner(getFile());
+        while (s.hasNext()) {
+            String line = s.nextLine();
+            TaskList task = new TaskList(line);
+            task.load();
+        }
+        s.close();
```
Maybe could consider laying out the code in a logical structure by splitting up code blocks with a space based on the logic? e.g could separate the initialisation of variables from line 48-50 and the while loop with a space

lambraydon reviewed (commented):
LGTM! Just some slight nitpicks in code quality.

wjacobw committed:
GUI feature

wjacobw committed:
Revert "GUI feature"

This reverts commit f34e4a1440d2e1df733e13972289cf0bd91aeb5b.

wjacobw committed:
adds GUI feature

wjacobw committed:
Merge branch 'branch-Level-10'

wjacobw committed:
Adds assertions to the TaskList Class

assertions is good to document important assumptions that should hold at
various points in the code.

wjacobw committed:
Clean the codes according to the Code Quality

There are some line of codes that are not used that might confuse the
reader

Deleting all the unused codes and some comments that can confuse readers

wjacobw committed:
Merge pull request #6 from wjacobw/A-Assertions

Adds assertions to the TaskList Class

wjacobw committed:
solves merge conflict

wjacobw committed:
Changes the code to adhere with coding standard

wjacobw committed:
changes ui

wjacobw committed:
adds user guide

wjacobw committed:
adds more ug

wjacobw committed:
changes data path

wjacobw committed:
modifies build.gradle

wjacobw committed:
more changes

wjacobw committed:
more changes

wjacobw committed:
last commit please
